.github/workflows/ci.yaml

```diff
@@ -494,14 +497,26 @@ jobs:
         run: |
           ./project/scripts/genDocs -doc-snapshot
 
-      - name: Deploy Website
+      - name: Deploy Website to dotty-website
         uses: peaceiris/actions-gh-pages@v3
         with:
-          personal_token: ${{ secrets.BOT_TOKEN }}
+          personal_token: ${{ DOTTY_WEBSITE_BOT_TOKEN }}
           publish_dir: docs/_site
           external_repository: lampepfl/dotty-website
           publish_branch: gh-pages
 
+      - name: Generate docs.scala-lang preprocessed files
+        run: |
+          ./project/scripts/genDocsScalaLang
+
+      - name: Deploy Website to docs.scala-lang
+        uses: peaceiris/actions-gh-pages@v3
+        with:
+          personal_token: ${{ DOCS_SCALALANG_BOT_TOKEN }}
+          publish_dir: docsScalaLang
+          external_repository: scala/docs.scala-lang
+          publish_branch: dev
```

Comment:
Just to confirm that I understood correctly, is this workflow going to push to scala/docs.scala-lang, on branch `dev`, the content of the documentation files stored here?